libs/functions/generate_api.py

```diff
@@ -7,14 +8,24 @@
 from libs.datasets import results_schema as rc
 
 
+def _format_date(input_date):
+    if not input_date: 
+        raise Exception("Can't format a date that doesn't exist")
+    if isinstance(input_date, str): 
+        # note if this is already in iso format it will be grumpy. maybe use dateutil
+        datetime_obj = datetime.strptime(input_date, '%m/%d/%Y %H:%M')  
```

Comment:
Where does this format type come from? would be great to make everything iso earlier in the flow.

there's also this new function that might be helpful: https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat

Reply:
ya it's in the JHU dataset we can format to ISO there although I think that PR is in flight @ghop02 wanna fix it in your PR?

Reply:
Yeah sounds good!